added diagrams

A-Faris opened the pull request:
Added the ERD and Architecture diagram
closes #7, closes #11

A-Faris committed:
added diagrams

A-Faris committed:
Merge branch 'main' into diagrams

FarihaChoudhury commented on diagrams/Architecture_Diagram.png:
I think the boxes seem inconsistent, there is no box for the SNS alert or the dashboard but there is for everything else

A-Faris replied:
Done

FarihaChoudhury commented on diagrams/ERD.png:
I think it would be easier to read if the tables were spaced out a little bit more.

A-Faris replied:
Done

A-Faris committed:
updated diagrams

A-Faris committed:
merging
Merge branch 'diagrams' of https://github.com/Zhi-704/Railway-Tracker into diagrams

A-Faris committed:
Merge branch 'main' into diagrams

FarihaChoudhury reviewed (approved):
Looks beautiful
